src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1520,6 +1520,17 @@ impl Coordinator {
                     }
                 }
                 ObjectId::Cluster(id) => {
+                    for id in self
+                        .catalog()
+                        .get_cluster(*id)
+                        .log_indexes
+                        .values()
+                        .cloned()
+                        .collect_vec()
+                    {
+                        self.drop_compute_read_policy(&id);
+                    }
```

Comment:
I think this is too early to drop the compute policies. We need to wait until after the catalog transaction has completed successfully before we can start sending messages to compute/storage.